lib/network_monitor.c

```diff
@@ -19,18 +19,22 @@
 #include <assert.h>
 
 #include <linux/rtnetlink.h>
+#include <arpa/inet.h>
 #include <net/if.h>  // For standard network interface flags.
 #include <netinet/in.h>
 
 #include <ell/netlink.h>
 #include <ell/log.h>
 #include <ell/util.h>
 #include <ell/queue.h>
+#include <ell/timeout.h>
+#include <ell/rtnl.h>
 
 #include <mptcpd/private/path_manager.h>
 #include <mptcpd/network_monitor.h>
 
-
+static struct in6_addr test_net_v6 = { .s6_addr = {0x20, 0x01, 0x0d, 0xb8, } };
```

Comment:
Maybe add comment to explain what this specific set of IPv6 addresses is?
```suggestion
// See IETF RFC 3849: IPv6 Address Prefix Reserved for Documentation.
// 2001:DB8::/32
static struct in6_addr const test_net_v6 = { .s6_addr = {0x20, 0x01, 0x0d, 0xb8, } };
```
Note the added `const`, too.